src/platform/nrfconnect/ConnectivityManagerImpl.h

```diff
@@ -17,6 +17,10 @@
 
 #pragma once
 
+#if !CHIP_DEVICE_CONFIG_ENABLE_THREAD && !CHIP_DEVICE_CONFIG_ENABLE_WIFI
+#error "At least one IPv6 networking backend must be enabled!"
```

Comment:
I think this will cause native_posix test failures. I suggest avoiding this assumption.

Reply:
Right, will remove it.